src/store/modules/projects.js

```diff
@@ -0,0 +1,24 @@
+import { firebaseAction } from 'vuexfire'
+import * as firebase from 'firebase'
+
+export default {
+	state: {
+		projects: [],
+		projectsRef : null
+	},
+	mutations: {
+		setProjectsRef:(state, payload) => {
+			state.projectsRef = payload
+		}
+	},
+	actions: {
+		projects: firebaseAction(({ commit, bindFirebaseRef}, {ref, callbacks})=>{
+			bindFirebaseRef('projects', ref, callbacks)
+			commit('setProjectsRef', ref)
+		})
+	},
+	getters: {
+		projects:state => state.projects
+		}
```

Comment:
The indentation of the file is not correct

Reply:
I will update it